dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1339,4 +1340,10 @@ private void validateVersusMinimumDate(FacesContext context, UIComponent toValid
             context.addMessage(toValidate.getClientId(context), message);
         }
     }
+    
+    public void datasetUnlockedInfo() {
```

Comment:
This function doesn't return information about unlocked dataset as name suggests, but creates info in order to show it. Imo it should be named something like "showDatasetUnlockedInfo".